test/lib/jdk/test/lib/net/IPSupport.java

```diff
@@ -154,4 +158,22 @@ public static void printPlatformSupport(PrintStream out) {
         out.println("preferIPv6Addresses: " + preferIPv6Addresses());
     }
 
+    /**
+     * Return current platform's maximum size for IPv4 UDP send buffer
+     */
+    public static final int getMaxUDPSendBufSizeIPv4() {
+        return IPV4_SNDBUF;
+    }
+
+    /**
+     * Return current platform's maximum size for IPv6 UDP send buffer
+     */
+    public static final int getMaxUDPSendBufSizeIPv6() {
+        if (Platform.isAix()) {
+            return IPV6_SNDBUF_AIX;
+        }
+        else {
```

Comment:
the right formatting here would be `} else {` (all in one line)

Reply:
still learning... thanks!